src/main/java/com/dev/cinema/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -41,4 +42,11 @@ public void registerNewShoppingCart(User user) {
         shoppingCart.setUser(user);
         shoppingCartDao.add(shoppingCart);
     }
+
+    @Override
+    public void clear(ShoppingCart shoppingCart) {
+        shoppingCart.setTickets(List.of());
```

Comment:
```suggestion
        shoppingCart.setTickets(Collections.emptyList());
```
It gives more information about what exactly you are doing